benchmarks/dbscan/ArborX_DBSCANVerification.hpp

```diff
@@ -306,7 +306,7 @@ bool verifyDBSCAN(ExecutionSpace exec_space, Primitives const &primitives,
   ARBORX_ASSERT(eps > 0);
   ARBORX_ASSERT(core_min_size >= 2);
 
-  Points points{primitives};
+  Points const &points{primitives};
```

Comment:
What warnings are here? This makes little sense to me. I would not want to change these.

Reply:
```
 warning: local copy 'points' of the variable 'primitives' is never modified; consider avoiding the copy [performance-unnecessary-copy-initialization]
```
We could just `NOLINT` here.

Reply:
Would using `Points const points{primitives};` work? If yes, I would prefer that to `NOLINT`.

Reply:
No, the point is that the copy could be avoided. `const` doesn't make a difference.

Reply:
Can we move `NOLINT` to inside `AccessTraits` somehow?

Reply:
I don't think so. The problem is that `Points` and `Primitives` could be the same type if the latter is a `Kokkos::View`. Another workaround would be
```C++
std::conditional_t<std::is_same_v<Predicates, UserPredicates>, Predicates const&, Predicates> predicates{user_predicates};
```
but I'd prefer NOLINT.